### DISCUSSION TITLE
RTÉ Jr

### DISCUSSION THREAD

**Discussion by M3UFreak** [iptv-org/community#717] [Q&A]
**Channel Name:** RTE JR
**Channel ID:** RTEjr.ie
**Country:** Ireland
**Language:** english
**Website:** rtejr.rte.ie
**Stream URL (search for the url or something.):** 
**Notes (optional):** (put it in HD plz)

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19748
